scripts/superdesk/editor2/editor.js

```diff
@@ -562,6 +562,13 @@ function SdTextEditorBlockEmbedController($timeout, editor, renditions) {
                 vm.onBlockChange();
             });
         },
+        isEditable: function(picture) {
+            if (picture._type === 'externalsource') {
```

Comment:
All search providers should set this because there is a logic on frontend for it.